oak-core/src/main/java/org/apache/jackrabbit/oak/plugins/index/AsyncIndexUpdate.java

```diff
@@ -426,6 +428,20 @@ private void checkIfStopped() throws CommitFailedException {
 
     @Override
     public synchronized void run() {
+        boolean isIndexWithLanePresent = false;
+        NodeState oakIndexNode = store.getRoot().getChildNode("oak:index");
```

Comment:
What happens if the node doesn't exist, does it still work? (In most cases, it exists of course, but in theory there might be no index... in which case it shouldn't throw an exception).

Reply:
added a check.